test/include/chroot_env.sh

```diff
@@ -27,7 +27,7 @@ setup_chroot_env()
     # overlayfs is required because we need to write data
     # in the ro squashfs for testing purposes,
     # e.g. ARM_EMU_BIN mount point and temporary test files.
-    overlayfs_dir="$(mktemp -d -t "overlayfs.XXXXXXXXXX")"
+    overlayfs_dir="$(mktemp -d -t "overlayfs.XXXXXX")"
```

Comment:
I think this was an unintended change whilst resolving.

Reply:
This is intended, six random positions are enough